PhysicsTools/PatAlgos/python/slimming/primaryVertexAssociation_cfi.py

```diff
@@ -4,10 +4,17 @@
 primaryVertexAssociation = sortedPFPrimaryVertices.clone(
   qualityForPrimary = cms.int32(2),
   produceSortedVertices = cms.bool(False),
-  producePileUpCollection  = cms.bool(False),  
+  producePileUpCollection  = cms.bool(False),
   produceNoPileUpCollection = cms.bool(False)
 )
 
+primaryVertexWithBSAssociation = primaryVertexAssociation.clone(
+  vertices = cms.InputTag("offlinePrimaryVerticesWithBS")
+)
+
 from Configuration.ProcessModifiers.run2_miniAOD_pp_on_AA_103X_cff import run2_miniAOD_pp_on_AA_103X
 run2_miniAOD_pp_on_AA_103X.toModify(primaryVertexAssociation,particles = "cleanedParticleFlow")
 primaryVertexAssociationCleaned = primaryVertexAssociation.clone(particles = "cleanedParticleFlow:removed")
+
+run2_miniAOD_pp_on_AA_103X.toModify(primaryVertexWithBSAssociation,particles = "cleanedParticleFlow")
+primaryVertexWithBSAssociationCleaned = primaryVertexWithBSAssociation.clone(particles = "cleanedParticleFlow:removed")
```

Comment:
is this really needed?

Reply:
Sincerely I placed it for consistency, for me we may erase it.

Reply:
@mandrenguyen @abaty could you please give your advice about the possible need of these `offlineVerticesWithBS` for HI?

Reply:
It would be nice to have offlinePrimaryVerticesWithBS in HI miniAOD as well.   I checked the increase in size on 100 events of wf 140.5611 and it looks negligible, around 0.1%.   PU is so low in PbPb that multiple vertices are more likely to come from vertex splitting.  So perhaps this is not unexpected.

Reply:
> It would be nice to have offlinePrimaryVerticesWithBS in HI miniAOD as well. I checked the increase in size on 100 events of wf 140.5611 and it looks negligible, around 0.1%. PU is so low in PbPb that multiple vertices are more likely to come from vertex splitting. So perhaps this is not unexpected.

Great, thank you Matt,
Therefore, this part can remain as it is,